cachesetup.sh

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+# This script sets up the CockroachDB databases and users for the Politeia
+# cache.  This script requires that you have already created all the necessary
+# CockroachDB certificates using the cachecerts.sh script and that you have a
+# CockroachDB instance listening on the default port localhost:26257.
+
+set -ex
+
+# CERTS_DIR must contain client.root.crt, client.root.key, and ca.crt for the
+# CockroachDB instance.
+readonly CERTS_DIR=$1
+
+# Database names.
+readonly DB_MAINNET="records_mainnet"
+readonly DB_TESTNET="records_testnet3"
+
+# Database user names.
+readonly USER_POLITEIAD="records_politeiad"
+readonly USER_POLITEIAWWW="records_politeiawww"
+
+if [ "${CERTS_DIR}" == "" ]; then
+    >&2 echo "Error: missing argument root user certs directory"
+    exit
+fi
+
+# Create the mainnet and testnet databases for the politeiad records cache.
+cockroach sql \
```

Comment:
'cockroach sql' needs to be a variable like ${SQL} that is set at the start of the script so that it can be replaced with psql.

Reply:
The cockroach cli flags don't map to the psql flags so that wouldn't work.  

Example: to execute a command with the cockroach cli you use `-e, --execute`, but to execute a command with psql you use `-c, --command`.

Reply:
If that is the case we need an equivalent script for psql. Got to admit that I really don't like having two scripts. Can the SQL at least overlap?